### PR TITLE
Update address for November

### PR DESCRIPTION
**Pull request by jarrettch**
Updating to the current address for November 4th.

### CODE REVIEW

File: data/index.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...going quickly.</p><h2 class=\"u-color--light\">Parking and Event time</h2><p class=\"u-color--light\">TBA</p><h2 class=\"u-color--light\">Check-in and Food / Drink</h2><p class=\"u-color--light\">Please find a name tag and write your name. Food and drink should be available so please help yourself before the event begins.</p><h2 class=\"u-color--light\">Tables and Wifi</h2><p class=\"u-color--light\">Please find a table and chair inside the main training room. Connect yourself to the wireless network and download any necessary programs and workshopper to get started (Node.js and Learnyounode if it's your first time with Node.js). The network name will displayed on the video screens. Mentors will be available to assist if you haven't done this step prior to arriving at the event.</p><h2 class=\"u-color--light\">What to Expect</h2><p class=\"u-color--light\">Actual time to get to work on a workshopper will be about 3 to 3 1/2 hours. You are free to work on any workshopper you like and mentors will be available if you get stuck or need any assistance. Everyone is here to learn so there are no egos and no questions that are too silly to ask. Come join and have fun while learning!</p>",
```
Comment (ashwell) [inline data/index.json]: there is a paragraph to add to this line, it is a total pain in the ass.

you can look at the diff of what I removed for guidance, https://github.com/nodeschool/los-angeles/commit/17d4e2f82d97c44d51ff28ad284db84259b8e462

Comment (jarrettch) [reply data/index.json]: Yeah I went based off the diff when I did this and somehow totally missed that lol. Nice catch.

Comment (ashwell) [reply data/index.json]: yeah, normally we don't have to change anything but the date so it is really simple.

this time is kinda weird

**Comment by jarrettch** [comment]
@rouzbeh84 if there are additional parking/guidance details you think we should add let me know, but I assume getting to the room is pretty straightforward with this info.

**Comment by rouzbeh84** [comment]
Should be good to go after either @ashwell or @jarrettch review minor updates

**Comment by jarrettch** [comment]
Should it be "in the structure"?

**Comment by rouzbeh84** [comment]
i will kill you haha. i mean whatever sounds good. its in a parking lot structure directly behind the building but you technically will most likely hit it first and the whole fuggin place is under construction. Its one of those things where during week its monitored but weekends not so yea..

**Comment by jarrettch** [comment]
I meant purely from a grammatical standpoint lol

**Pull request merged**